includes/utils.php

```diff
@@ -653,6 +653,17 @@ public static function is_empty( $source, $key = false ) {
 		return '0' !== $source && empty( $source );
 	}
 
+	/**
+	 * Checks whether a control is a tablet/mobile setting generated by add_responsive_control()
+	 *
+	 * @param array $control
+	 *
+	 * @return bool
+	 */
+	public static function is_control_responsive_generated( $control ) {
```

Comment:
Until we find a better place, do this check inline and don't create a new method